lib/sync/fetcher.go

```diff
@@ -22,6 +23,7 @@ import (
 	"boscoin.io/sebak/lib/transaction"
 
 	"github.com/inconshreveable/log15"
+	pkgerrors "github.com/pkg/errors"
```

Comment:
I think it would be better to import this in sebak's `error` package and expose `Wrap` from there.

Reply:
Ok,